simple/stats/importer.py

```diff
@@ -136,22 +116,41 @@ def remove_pre_resolved(entity: str) -> bool:
             unresolved=unresolved_list,
         )
 
+    def _resolve(self, entity_column_name: str,
+                 entities: list[str]) -> dict[str, str]:
+        if entity_column_name.casefold(
+        ) == constants.INPUT_COLUMN_DCID.casefold():
```

Comment:
very nit, can use a function like check_column(column_name, constatns.xxx) and use casefold() in the function.

Reply:
I've made the logic more generic. PTAL.